php/README.md

```diff
@@ -0,0 +1,25 @@
+# Custom Auth with php
+
+The core logic is in `Ionic\CustomAuthenication` class which can be subclassed
```

Comment:
I'm sorry my desire to abstract and have small functions got in my way of offering an _example_. I will rewrite to match the other examples by making it very functional. Then I'll start working on my unofficial php SDK library... It really can be very simple if we agree on a architecture.